simplified-books-core/src/main/java/org/nypl/simplified/books/core/BooksControllerRevokeBookTask.java

```diff
@@ -539,6 +558,10 @@ private void revokeLoanedWithDRM(final URI revoke_uri)
         if (in_error.startsWith("E_ACT_NOT_READY")) {
           this.error = Option.some((Throwable) new AccountNotReadyException(in_error));
         }
+        else if (in_error.startsWith("E_STREAM_ERROR")) {
```

Comment:
This has not yet been approved in a separate PR I have in to Master, so remove this change for now.